guidelines/index.html

```diff
@@ -2665,12 +2665,12 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </dd>
 
             				
-            <dt class="new"><dfn data-lt="dragging movements|dragging movement" id="dfn-dragging-movements" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">dragging movement</dfn></dt>
+            <dt class="new"><dfn data-lt="dragging movements|dragging movement" id="dfn-dragging-movements" tabindex="0" aria-haspopup="dialog" data-dfn-type="dfn">ドラッグ動作 (dragging movement)</dfn></dt>
 <dd class="new"><p class="change">[New]</p>
    					
-  <p>an operation where the pointer engages with an element on the <a href="#dfn-down-event" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-down-event-2" title="トリガ刺激が depressed (押し下げられた) となった時に発生するプラットフォームのイベント。">down-event</a> and the element (or a representation of its position) follows the pointer until an <a href="#dfn-up-event" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-up-event-2" title="ポインタのトリガが解除されたときに生じるプラットフォームイベント。">up-event</a></p>
+  <p><a href="#dfn-down-event" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-down-event-2" title="トリガ刺激が depressed (押し下げられた) となった時に発生するプラットフォームのイベント。">ダウンイベント</a>と同時にポインタが要素とつながり、<a href="#dfn-up-event" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-up-event-2" title="ポインタのトリガが解除されたときに生じるプラットフォームイベント。">アップイベント</a>になるまでその要素 (又はその位置の表現) がポインタに追従する操作。</p>
```

Comment:
engages を 「つながる」とするのは微妙に違和感があります。
マウスと要素が相互に関連し、同時に動くようになるという話なのですが、「つながる」と言われると物理的な、あるいは可視のつながりが発生するような感じがします (個人的な語感ですが)
たとえば「結びつく」というくらいではどうでしょうか?

```suggestion
  <p><a href="#dfn-down-event" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-down-event-2" title="トリガ刺激が depressed (押し下げられた) となった時に発生するプラットフォームのイベント。">ダウンイベント</a>と同時にポインタが要素と結びつき、<a href="#dfn-up-event" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-up-event-2" title="ポインタのトリガが解除されたときに生じるプラットフォームイベント。">アップイベント</a>になるまでその要素 (又はその位置の表現) がポインタに追従する操作。</p>
```

Reply:
実はこの部分も、翻訳していて、違和感が残っていました。なるほど「結びつく」ですね。よいと思います！